### ISSUE TITLE
Can not use github image as main screenshot

### ISSUE THREAD

**Issue by DelmorS** [userstyles-world/userstyles.world#81]
When trying to add main screenshot to my style, it says, that you can use github storage for it.
But when I trying to set following screenshot:

https://github.com/DelmorS/The-Best-Google-Logo---My-Little-Pony-MLP-Logo/blob/master/docs/images/main_page_examples.png

Textbox is turning red, and tells, that I can not use this link.

And the same goes to direct link:
https://github.com/DelmorS/The-Best-Google-Logo---My-Little-Pony-MLP-Logo/raw/master/docs/images/main_page_examples.png

Is it possible to fix it, or to add some tip for user, why it can not be used, or how to fix url

<img width="559" alt="Screenshot_1" src="https://user-images.githubusercontent.com/41177486/129303043-a06d563f-ace1-4b6c-ae13-f07df7909394.png">

Thank you.

**Comment by Gusted** [comment]
Hi @DelmorS!

Those links are actually redirection links to the correct URL([raw.githubusercontent.com](https://raw.githubusercontent.com/DelmorS/The-Best-Google-Logo---My-Little-Pony-MLP-Logo/master/docs/images/main_page_examples.png)) when you click on them, you will need to use those URL as value.

Regards,
Gusted

**Comment by vednoc** [comment]
I deployed the fix to production. The issue will close automatically once `dev` branch is merged into `main`.

Thank you for reporting this bug. 🎉
